Makefile

```diff
@@ -147,18 +147,34 @@ build_ui:
 build-all-in-one-linux: build_ui
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/standalone/standalone-linux $(BUILD_INFO) ./cmd/standalone/main.go
 
+.PHONY: build-all-in-one-windows
+build-all-in-one-windows: build_ui
+	CGO_ENABLED=0 GOOS=windows installsuffix=cgo go build -o ./cmd/standalone/standalone-windows.exe $(BUILD_INFO) ./cmd/standalone/main.go
+
 .PHONY: build-agent-linux
 build-agent-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/agent/agent-linux $(BUILD_INFO) ./cmd/agent/main.go
 
+.PHONY: build-agent-windows
+build-agent-windows:
+	CGO_ENABLED=0 GOOS=windows installsuffix=cgo go build -o ./cmd/agent/agent-windows.exe $(BUILD_INFO) ./cmd/agent/main.go
```

Comment:
Any reason not to call this `agent.exe`? Is there any other platform that uses `.exe` extension?

Reply:
Just following convention since the agent for linux is called `agent-linux`. I'll change it up to be `agent.exe`.

Reply:
@yurishkuro what if we named the different binaries for windows along the lines of `jaeger.<product>.exe` and dropped the `-windows` on all of them?

Reply:
my preference would be to have identical names but group them into dirs/archives by target, e.g.
```
jaeger-1.4.0-darwin-amd64.tar.gz
  - agent
  - collector
  - etc
jaeger-1.4.0-linux-amd64.tar.gz
  - agent
  - collector
  - etc
jaeger-1.4.0-windows-amd64.tar.gz
  - agent.exe
  - collector.exe
  - etc
```

Reply:
I like it. I'll work on getting that setup @yurishkuro.